README.md

```diff
@@ -327,6 +327,46 @@ index
 controller.abort()
 ```
 
+
+### Using Meilisearch behind a proxy <!-- omit in toc -->
+
+#### Custom request config <!-- omit in toc -->
```

Comment:
In both `Custom request config` and `Custom http client` can you change the indentation to 2 spaces. Some are 4 spaces and some are missing indentation..